### DISCUSSION TITLE
[CHANGE] Desert Path Fix

### DISCUSSION THREAD

**Discussion by bayselb** [sunflower-land/sunflower-land#4579] [Ideas]
- Problem
Ever since the desert island update was released, I have noticed that paths have become unattractive to look at.
I think this issue (which is an easy fix) is due to the paths underneath the buildings now sit against the new paths. 

- Solution 
Redesign desert path artwork 
Remove paths from underneath buildings 

![screenshot_desert_path_unfixed](https://github.com/sunflower-land/sunflower-land/assets/133677194/a97a0f8f-bb60-42f3-93ca-0e58711f7b39)

**Comment by bayselb** [comment]
Here is the new path example
![desert_island_path_example_large](https://github.com/sunflower-land/sunflower-land/assets/133677194/53258be8-a906-4733-8f43-ff5f5a704f08)

**Comment by bayselb** [comment]
![desert_path](https://github.com/sunflower-land/sunflower-land/assets/133677194/0200c36a-2c49-46c2-ae9d-1dd483400159)

**Comment by Polysemouse** [comment]
To implement this, we need all 16 possible configurations of the sprite.  For example, these are what the dirt ones looks like
![0_0_0_0](https://github.com/sunflower-land/sunflower-land/assets/107602352/c50e986d-ba34-4c7b-acc9-12745fba0e47)
![0_0_0_1](https://github.com/sunflower-land/sunflower-land/assets/107602352/d796029c-82e8-4826-9212-1c9ef2367e16)
![0_0_1_0](https://github.com/sunflower-land/sunflower-land/assets/107602352/6eeca5de-2cc8-4e1a-9690-6797e0b9fba1)
![0_0_1_1](https://github.com/sunflower-land/sunflower-land/assets/107602352/af3432f9-25a2-418d-b009-e86db4335000)
![0_1_0_0](https://github.com/sunflower-land/sunflower-land/assets/107602352/a69649d6-493b-4e82-ae43-cd6c4928b3ea)
![0_1_0_1](https://github.com/sunflower-land/sunflower-land/assets/107602352/30b040ed-3e3b-4b64-9f69-9ff4ceb1f85d)
![0_1_1_0](https://github.com/sunflower-land/sunflower-land/assets/107602352/ca864870-24bd-4d97-83f1-bca6c9c7dc59)
![0_1_1_1](https://github.com/sunflower-land/sunflower-land/assets/107602352/576a55d0-f8c0-44f0-9496-bff2ca0a99ec)
![1_0_0_0](https://github.com/sunflower-land/sunflower-land/assets/107602352/96335b81-1c61-4d00-9386-8d8182dfc15b)
![1_0_0_1](https://github.com/sunflower-land/sunflower-land/assets/107602352/b1c0c171-ee15-4d43-be17-7e01dd1c9b41)
![1_0_1_0](https://github.com/sunflower-land/sunflower-land/assets/107602352/29e2c54a-22a8-4588-b6d7-e8d896b601f6)
![1_0_1_1](https://github.com/sunflower-land/sunflower-land/assets/107602352/3af2e0b5-34c8-45f8-8420-fe8325a7d15e)
![1_1_0_0](https://github.com/sunflower-land/sunflower-land/assets/107602352/ce1904a3-eeb5-465b-9f72-37add0c0bd24)
![1_1_0_1](https://github.com/sunflower-land/sunflower-land/assets/107602352/6043dcf2-2f22-4083-bcb4-f85fae5a95d7)
![1_1_1_0](https://github.com/sunflower-land/sunflower-land/assets/107602352/6f0f8b5a-fc91-4323-bace-52ac077fc260)
![1_1_1_1](https://github.com/sunflower-land/sunflower-land/assets/107602352/3ffdfab4-b2f1-4e34-8539-d22b9ecf7042)
